python/paddle/fluid/tests/unittests/ngraph/test_activation_ngraph_op.py

```diff
@@ -18,17 +18,7 @@
 import numpy as np
 import paddle.fluid.core as core
 from paddle.fluid.tests.unittests.op_test import OpTest
-from paddle.fluid.tests.unittests.test_activation_op import TestRelu, TestTanh
-
-
-class TestNGRAPHReluDim2(TestRelu):
-    def setUp(self):
-        super(TestNGRAPHReluDim2, self).setUp()
-
-
-class TestNGRAPHTanhDim2(TestTanh):
-    def setUp(self):
-        super(TestNGRAPHTanhDim2, self).setUp()
```

Comment:
where is test sigmoid?

Reply:
The TestSigmoid is imported

Reply:
seems added `TestNGRAPHReluDim4` and `TestNGRAPHTanhDim4`, but no sigmoid one?

Reply:
The TestSigmoid test case was imported. It will run as part of the unittest through ngraph. It was derived to another testcase as no change is needed from the base except it will run through ngraph.